Documentation/release-notes/5071.md

```diff
@@ -0,0 +1,7 @@
+#### Application and library build and deployment
+
+  * [Developer Community 1141659][0]: Fixes an issue where including a
+    `.DS_Store` or any file starting with `.` as an `AndroidResource`
+    could cause application builds to hang indefinitely.
```

Comment:
To provide an example of the user-visible symptoms, include the regression information, and mention the IDE terminology "Build Action," maybe:

```markdown
   * [Developer Community 1141659][0]: Starting in Xamarin.Android 11.0,
     the *Updating Resources...* and *Cleaning solution* build steps
     could get stuck indefinitely if `.DS_Store` or any file starting
     with `.` was included in the project with the **AndroidResource**
     Build Action.
```

Reply:
Looks good, I don't think this would actually cause `Clean` to hang because it doesn't run `aapt2` at all, so I left that out.

Reply:
Sweet, and good point about cleaning solution. Not sure what was happening for the user on the Developer Community issue in that case, but I wouldn't be surprised if it was just an odd side-effect of the earlier problem when building that'll all be taken care of now. Anyway, sounds good to me on leaving out the `*Cleaning solution*` part from the release note :+1: